pom.xml

```diff
@@ -70,8 +75,8 @@
 		<!--Force UTF-8 for the build -->
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 		<staging.dir>${project.build.directory}/staging</staging.dir>
-		<jmockit-version>1.5</jmockit-version>
-		<graalvm.version>22.3.5</graalvm.version>
+		<jmockit-version>1.7</jmockit-version>
```

Comment:
Sorry, but I have pushed to master the PR #324, which introduces a conflict. Can I let you handle the conflict? Don't hesitate to ask if you have any doubt about the resolution.